docs/manual/en/manual.txt

```diff
@@ -0,0 +1,297 @@
+Note that there is no "0" setting; even at the lowest level, the operator can be heard faintly.
+
+The frequency multiplier "F*" controls the base frequency of the operator, in octaves.
+
+The key scale level "Ksl" determines the extent to which amplitude is attenuated as pitch increases. The parameter value maps to attenuation as follows:
+
+- "0" means no attenuation by frequency;
+- "1" attenuates by 3.0 decibels per octave;
+- "2" attenuates by 1.5 decibels per octave;
+- "3" attenuates by 6.0 decibels per octave.
+
+Note values "1" and "2" carefully.
+In the OPL3 chip, attenuation does not increase monotonically with respect to the parameter value.
+
+=== Other parameters
+
+The remaining parameters fall into two groups: Miscellaneous and Global.
+
+A miscellaneous parameter applies to the currently selected program in the editor. On the other hand, a global parameter is a property of the bank, and applies to all programs in that bank.
+
+.Miscellaneous parameters
+image::../resources/misc.png[200,200]
+
+"Velocity Offset" is a fixed modifier applied to the velocity of MIDI notes, and can be set in the range [-127, 127]. A positive offset increases velocity, and a negative value decreases it.
+
+.Global parameters
+image::../resources/global.png[230,230]
+
+The "Deep tremolo" and "Deep vibrato" flags make those effects more pronounced, if they are active. The effect depths are fixed values determined by the design of the YMF262, and are as follows:
+
+- Regular tremolo modulates amplitude by 1.0 dB
+- Deep tremolo modulates amplitude by 4.8 dB
+- Regular vibrato modulates frequency by 7 cents (7 hundredths of a semitone)
+- Deep vibrato modulates frequency by 14 cents
+
+"Volume model" specifies a function for transforming the discrete parameters associated with volume into actual gain values. In most cases, the generic model is appropriate.
+
+However, when loading banks from other software, the formula that reproduces the operation of that software should be selected. For example, if you load a bank from Doom, you should choose the DMX model. For Duke Nukem 3D, use Apogee.
+
+== About
+
+ADLplug was developed by Jean Pierre Cimalando in 2018. +
+It is based on previous work by several groups of people.
+
+The YMF262 emulators were developed by:
+
+- The DOSBox team.
+- Alexey Khokholov (Nuke.YKT) for Nuked OPL, and Alexander Troosh for his optimizations on v1.7.4.
+- Robson Cozendey for the original Java OPL, and Vitaly Novichkov (Wohlstand) for the C++ version.
+- 90's demo group Reality for Opal.
+
+The MIDI driver was previously developed by Joel Yliluoma (Bisqwit). Originally, this program, ADLMIDI, was a simple MIDI file player.
+
+Vitaly Novichkov (Wohlstand) reorganized ADLMIDI into a reuseable library, adding the basic building blocks of realtime MIDI control. Jean Pierre Cimalando has contributed improvements which make it more suitable for usage in real-time composition.
```

Comment:
These are natural:
- "contributed improvement**s** which make it more suitable"
- "contributed an improvement which make**s** it more suitable"

I had to Google it, but I *think* it's because 'which' is a pronoun:
https://ell.stackexchange.com/questions/27163/thats-what-make-makes-the-beautiful-rainbow
I never considered it before, but plural vs singular in English is very complicated and unintuitive :thinking:

Reply:
no worries, I might be wrong here

Reply:
I think the current form is fine.